step-ca/provisioners.mdx

```diff
@@ -678,6 +678,8 @@ Example `ca.json` provisioner configuration for a Google provisioner:
 
 - **listenAddress**<Reference id="star3" marker="*" />: is the address (`:port` or `host:port`) where the authorization server will redirect the client's web browser at the end of the authorization flow. By default, the `step` client will bind to 127.0.0.1 on a random port. This parameter is only required if the authorization server demands a specific port for loopback IP redirect URIs.
 
+- **scopes**: a space-separated list of scopes OAuth clients should request. This is used by the `step` CLI. The default value is `openid email`.
```

Comment:
In the provisioner configuration are meant to be a list of strings. See https://pkg.go.dev/github.com/smallstep/certificates@v0.28.3/authority/provisioner#OIDC

```go
Scopes                [][string](https://pkg.go.dev/builtin#string) `json:"scopes,omitempty"`
```

But if you add just one member separated by empty spaces that also works. If an array is passed, the cli will take care of joining them. And in the `step oauth` command the --scope flag can be passed multiple times to pass multiple scopes too.